### DISCUSSION TITLE
Clicking "Notifications" icon on GitHub Gist leads to 404

### DISCUSSION THREAD

**Discussion by PythonCoderAS** [community/community#20073] [Other Feature Feedback, Questions, & Ideas]
If I click the notification bell on any GitHub Gist webpage, such as https://gist.github.com/, I go to a 404 since the link is **https://gist.github.com/notifications**.

**Comment by andypiper** [comment]
Ah! Yes. I wanted to report this as well... I cannot see a repo for "gist" under github.com/github so I was wondering where is best to report it. It's slightly annoying, as I see a blue dot and I guess it is picking up the Notifications status from the main GitHub domain, but there's no notification page for gist.github.

**Comment by radar-blip** [comment]
Several others have reported this same issue and filed bug reports, hopefully it will be fixed soon. 
Just letting you know.
Have a great day!

You can report it as a bug here:  https://support.github.com/contact/bug-report
